tests/test_plugin.py

```diff
@@ -63,6 +63,23 @@ def test_build_plugin_failure(docker_tasker):
         results = runner.run()
     assert workflow.build_process_failed is True
 
+@pytest.mark.parametrize('required', [
+    True,
+    False,
+])
+def test_not_required_prebuild_plugin_failure(docker_tasker, required):
+    workflow = DockerBuildWorkflow(SOURCE, "test-image")
+    assert workflow.plugin_failed is False
+    runner = PreBuildPluginsRunner(docker_tasker, workflow,
```

Comment:
They don't now, but this test should protect from all possible situations.

Probably this is worth doing in next PRs for this test